demo/plt/rcv1x_2K.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+import os
+import time
+
+# Parameters
+k = 2456
+kary_tree = 16
+l = 1.0
+passes = 3
+b = 28
+output_model = "rcv1x_model"
+
+# Dataset filenames
+train_data = "rcv1x_train.vw"
+test_data = "rcv1x_test.vw"
+
+print("Vowpal Wabbit PLT example on RCV1X-2K dataset with {} labels".format(k))
+
+# Download dataset
+if not os.path.exists(train_data):
+    os.system("wget http://www.cs.put.poznan.pl/mwydmuch/data/{}".format(train_data))
```

Comment:
Perhaps a comment here with a link to the original source of data and how it was converted to vw format?
Specific command would be helpful for conversion would be helpful.

Reply:
unless this is optional, we should limit pulling in info during build or normal setup. This should be an opt-in. Haven't read if demo dir is built or just a basic demo directory.

Reply:
I agree with you, but I think it is okay in this directory. Just a demo dir that isn't built.